### ISSUE TITLE
Kal Vas Flam

### ISSUE THREAD

**Issue by ummahusla** [ultimaonline/server#1]
When it's going to be ready?

**Comment by gokaygurcan** [comment]
Hello @ummahusla,

I'm really happy to see someone finally interested in this repository and asked the BIG question. 

I'm working on this project during my spare time(s) and in these days I rarely find one. But it looks like you read&write Javascript and playing video games, especially Ultima Online. So, we can decide about the structure together and try to follow it as much as we can. 

So far, I was only able to send/receive packets and let player log in to the game. After that, I needed to have a working game environment to send out the correct data packages (i.e. walking, clicking, talking, etc.) and I couldn't have much time to have all these stuff. 

My initial idea was to start working on the `ultimaonline/server` repository, and then move stuff out to their own repositories like `ultimaonline/player`, `ultimaonline/mobile`, `ultimaonline/item`, etc. 

PLEASE share what you thing and how it should be with me, and then maybe I can speed up a bit. 

Thanks!

**Comment by ummahusla** [comment]
What is your global idea about that project? 

Additionally, do you know that Ultima Online is a trademark of Electronic Games and taking it as an organisation name might be a bit of the risk?

Next month it's starting a [GitHub's game jam](https://github.com/blog/2262-github-s-game-jam-game-off-returns-next-month), so we can think maybe we can do something for this purpose.

**Comment by gokaygurcan** [comment]
The initial idea was to replace @sphereserver and @runuo with something more up to date. This might seem a little bit weird but after all these years with Sphere and its meaningless scripting language, I'd prefer something like javascript in any case. And since RunUO project ended (!) there's no good alternative to these two. 

I never planned to make some profit out of this, this is purely for fun. 

And yes, I know that the name belongs to EA but it's not that much hard to change the name if they want to take it. I already mailed them, I explained what I'm planning to do and asked that if this has any legal risks or something, and after more than a year there's no reply or anything. 

About the Game Jam thingy, it says "contestants have 1 month to build an open source game that is playable in the web browser." and I never planned to make this work on a browser. I think it'd be limiting the abilities of the game and the player but of course this is my opinion. The reason that I chose `server` as the repository name was maybe one day the community can start working on a `client` project, to eliminate TEOT kind of hacks and everything.
